compiler_opt/tools/generate_default_trace_test.py

```diff
@@ -59,10 +59,27 @@ def collect_data(self, module_spec, tf_policy_path, reward_stat):
         keys=['default'])
 
 
+class MockProblemConfiguration():
+  """A problem configuration for testing."""
+
+  def flags_to_add(self):
+    return []
+
+  def flags_to_delete(self):
+    return []
+
+
+def mock_get_configuration():
+  return MockProblemConfiguration()
+
+
 class GenerateDefaultTraceTest(absltest.TestCase):
 
+  @mock.patch('compiler_opt.rl.registry.get_configuration')
```

Comment:
what is the problem if you don't have the mocking here?

Reply:
It will error out as `registry.get_configuration` relies on being configured through gin and it doesn't get configured by pytest, so we need to mock it so that it doesn't rely on being configured through a gin file.

Reply:
you can have gin configs following the code in line 112~115

i have the runner mock here because we don't want to really run compilation here; for you change, you don't need the mock